src/hair-landing/hair-landing.html

```diff
@@ -0,0 +1,58 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://kit.fontawesome.com/4fa7e132c9.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="iconsheet" href="all.css">
```

Comment:
this icon css needs to load before you can use it in your styles.css file so move this above it.